modules/xfem/doc/content/modules/xfem/theory/theory.md

```diff
@@ -1,4 +1,127 @@
+
+!media xfem/virtual_edge_merge.png
+       id=virtual_edge_merge
+       style=width:50%;margin-right:auto;margin-left:auto;
+       caption=Scenario in which the unmodified element fragment algorithm results in extra degrees of freedom. Without special treatment, there are two coincident phantom nodes at the indicated location because the neighboring edges share no physical material.
+
+## Partial Element Integration Algorithm
+
+Aside from the mesh cutting algorithm, the other major modification that must be made to the finite element code for the phantom node algorithm is that quadrature rules that are used to perform volume integrals, such as for computing the integral of the divergence of the stress tensor to compute an element's contribution to the residual, are adjusted to integrate over the physical portions of the cut elements. This is commonly done by forming a set of triangular (2D) or tetrahedral (3D) sub-elements that cover the entire physical portion of the split element's domain. These sub-elements are used only for integration, and new integration points are introduced at the locations of those sub-elements' integration points.
+
+A significant drawback of this sub-division approach to partial element integration is that when there is stateful material data at integration points (as is often the case for nonlinear mechanical constitutive models and a variety of other behavior models used in fuel performance analysis), this data must be mapped to the new integration points. There are approaches to integration of partial elements that avoid this mapping of material data to new integration points by using the original integration points, but scaling the weight factors in the numerical integration procedure to give a reasonable approximation of the integration of a function over a portion of the domain. The simplest approach is to simply scale the weights by a factor equal to the physical volume fraction of a split element. A more sophisticated approach known as moment fitting uses a least squares procedure to compute separate weights for each integration point to optimally integrate a specified set of functions. A recent study by [!cite](Zhang2018) shows on a variety of problems that the moment fitting approach gives results that are slightly improved over the simple volume fraction approach, and that both approaches are viable on problems of practical interest. The moment fitting scheme described in that work is used with a four-point integration rule for the 2D linear quadrilateral elements used in the present study.
```

Comment:
References for original quadrature weighting method?

Reply:
We cited it somewhere else.

Reply:
Just out of curiosity, do you have the reference? I think I went looking for it when doing some thesis writing the other day and I couldn't find it :)